Apply clang-format to RPC files

mrshenli opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#34139 Apply clang-format to RPC files**
* #34095 Use double quotes in C++ to stay consistent with Python RPC docs
* #34081 Improve ProcessGroup RpcBackendOptions Constructor API

Differential Revision: [D20227342](https://our.internmc.facebook.com/intern/diff/D20227342)

mrshenli committed:
Apply clang-format to RPC files

[ghstack-poisoned]

mrshenli committed:
Update on "Apply clang-format to RPC files"




[ghstack-poisoned]

rohan-varma reviewed (approved):
Thanks! Python lint error is unrelated.

mrshenli committed:
Update on "Apply clang-format to RPC files"


Differential Revision: [D20227342](https://our.internmc.facebook.com/intern/diff/D20227342)

[ghstack-poisoned]

mrshenli committed:
Update on "Apply clang-format to RPC files"


Differential Revision: [D20227342](https://our.internmc.facebook.com/intern/diff/D20227342)

[ghstack-poisoned]

mrshenli committed:
Update on "Apply clang-format to RPC files"


Differential Revision: [D20227342](https://our.internmc.facebook.com/intern/diff/D20227342)

[ghstack-poisoned]

mrshenli committed:
Update on "Apply clang-format to RPC files"


Differential Revision: [D20227342](https://our.internmc.facebook.com/intern/diff/D20227342)

[ghstack-poisoned]
